### PR TITLE
Update NFT rewards design

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

1. Fixes images to squares (centered) for all thumbnails site-wide. Can still see full image by clicking 'preview' in most places. Need to add a preview button to the card on project page (#1692)

2. Reducing spacing between cards on mobile.

https://user-images.githubusercontent.com/96150256/183336950-abf91ae1-a7e1-4643-b145-17407e15e25f.mp4

3. Removes locks from cards (user-feedback deemed it confusing)

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "prepare": "husky install",
     "i18n:extract": "./scripts/extract-translations-template.sh",
     "i18n:extract-source-locale": "NODE_ENV=development lingui extract --locale en",
-    "i18n:compile": "NODE_ENV=development yarn i18n:extract-source-locale && lingui compile",
```
Comment (johnnyd-eth) [inline package.json:83]: Not sure why I'm getting a diff here 🤔

File: src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx [hunk 2]
```diff
@@ -48,8 +51,10 @@ export default function NftUpload({ form }: { form: FormInstance }) {
   const imageUrl = form.getFieldValue('imageUrl')
 
   const validateImageUrl = () => {
-    if (imageUrl === undefined) {
-      return Promise.reject()
+    if (uploading) {
+      return Promise.reject('File uploading.')
```
Comment (johnnyd-eth) [inline src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx:55]: Show error message when trying to submit new NFT reward while the image is still uploading

File: src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx [hunk 3]
```diff
@@ -89,6 +94,8 @@ export default function NftUpload({ form }: { form: FormInstance }) {
         />
       }
       rules={[{ required: true, validator: validateImageUrl }]}
+      extra={t`Image will be cropped to a square in thumbnail previews through the Juicebox app.`}
```
Comment (tomquirk) [inline src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx:97]:
```suggestion
      extra={t`Image will be cropped to a square in thumbnail previews on the Juicebox app.`}
```

**Review by tomquirk — CHANGES_REQUESTED** [review]
See inline comments

**Review by tomquirk — COMMENTED** [review]
@johnnyd-eththis LGTM, just need to fix up that package.json diff

File: src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx (hunk 3 above)
Comment (johnnyd-eth) [reply src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx]: Yep done.

File: package.json (hunk 1 above)
Comment (johnnyd-eth) [reply package.json]: Fixed

**Review by tomquirk — APPROVED** [review]
@johnnyd-eth LGTM!!

**Pull request merged**